tests/lax_scipy_sparse_test.py

```diff
@@ -109,14 +133,59 @@ def args_maker():
         args_maker,
         tol=2e-2)
 
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+          "_shape={}_preconditioner={}".format(
+              jtu.format_shape_dtype_string(shape, dtype),
+              preconditioner),
+          "shape": shape, "dtype": dtype, "preconditioner": preconditioner}
+      for shape in [(4, 4), (7, 7), (32, 32)]
+      for dtype in float_types + complex_types
+      for preconditioner in [None, 'identity', 'exact']))
+  def test_bicgstab_against_scipy(self, shape, dtype, preconditioner):
+
+    rng = jtu.rand_default(self.rng())
+    A = poisson(shape, dtype)  # use scipy's test to check bicgstab instead of random matrix
+    # A = rand_sym_pos_def(rng, shape, dtype)
```

Comment:
I would be in favor of testing the Poisson equation _as well_ as random matrices, but I would definitely test random matrices (with no structure) as well. This seems much more likely to turn up issues.

Reply:
Yeah, I agree, I thought of this as well. I will implement that suggestion!